docs/content/en/docs/snippets/tasks/install.md

```diff
@@ -1,7 +1,28 @@
 
 ## Install version 0.6.0 and above
 
-In version 0.6.0 and later, you can install the Lifecycle Toolkit using the current release manifest:
+In version 0.6.0 and later, you can install the Lifecycle Toolkit using helm charts:
+
+```
+helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn-lifecycle-toolkit keptn/lifecycle-toolkit -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+To install a specific version use `--version <version>` as part of `helm upgrade --install` command.
+
+To list available versions:
+
+```
+helm repo update
+helm search repo keptn-lifecycle-toolkit
+```
+
+The `helm upgrade --install` command offers a flag called `--set`, which can be used to specify configuration options using the format key1=value1,key2=value2,....
+The full list of available flags can be found in the [helm-charts](https://github.com/keptn/lifecycle-toolkit/blob/main/helm/chart/README.md).
+
+For installing the Lifecycle Toolkit via manifests use:
```

Comment:
```suggestion
All versions of the Lifecycle Toolkit can be installed using manifests, with a command like the following:
```

Reply:
What would you think about moving the info about installing from manifests under the 0.5.0 then adding a note that later versions can also use this method?